coreUtilities/src/main/java/org/opensearch/migrations/utils/ArgLogUtils.java

```diff
@@ -0,0 +1,32 @@
+package org.opensearch.migrations.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArgLogUtils {
+
+    private ArgLogUtils() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static final String CENSORED_VALUE = "******";
+
+    public static List<String> getRedactedArgs(String[] args) {
+        List<String> redactedArgs = new ArrayList<>();
+        boolean shouldCensorNext = false;
+
+        for (String arg : args) {
+            if (shouldCensorNext) {
+                redactedArgs.add(CENSORED_VALUE);
+                shouldCensorNext = false;
+            } else if (arg.equals("--target-password") || arg.equals("--targetPassword")) {
```

Comment:
These should be passed in.  We may have passwords for work coordination or sources too.